.goreleaser.yaml

```diff
@@ -0,0 +1,113 @@
+project_name: rill
+
+release:
+  prerelease: auto
+  draft: true
+
+builds:
+  - id: darwin_amd4
+    main: ./cli/main.go
+    binary: rill
+    goos:
+      - darwin
+    goarch:
+      - amd64
+    env:
+      - PKG_CONFIG_SYSROOT_DIR=/sysroot/macos/amd64
+      - PKG_CONFIG_PATH=/sysroot/macos/amd64/usr/local/lib/pkgconfig
+      - CC=o64-clang
+      - CXX=o64-clang++
+    flags:
+      - "-mod=readonly"
+    ldflags:
+      - -s -w -X main.Version={{.Version}} -X main.Commit={{.ShortCommit}} -X main.BuildDate={{.Date}}
```

Comment:
I tested this, and noticed that running `rill version` outputs the correct version and commit hash, but for the build date, it outputs the current time, not the build time. Can you fix this? I think it might be because of the assignment in `main.go`, but not sure – maybe just have it be the zero time (`var BuildDate time.Time`) instead of `time.Now()`

Reply:
Yeah it is because of assignment in main.go